### DISCUSSION TITLE
Ancestry support misinformation on GEDCOM7 compatibility

### DISCUSSION THREAD

**Discussion by emyoulation** [FamilySearch/GEDCOM#305] [Q&A] [answered]
Today an Ancestry user in Germany received a support reply about a GEDCOM export from Ancestry that the reason Gramps couldn't parse the non-standard "d.m.yyyy" formatted dates in their file was that Ancestry wrote 7.0.11 version GEDCOM files. And that there was no option to export older versions anymore.

Of course, the exported identifies as version 5.5.1 in a test file header that was exported today:

```
0 HEAD 
1 SUBM @SUBM1@ 
1 SOUR Ancestry.com Family Trees 
2 NAME Ancestry.com Member Trees 
2 VERS 2022.08 
2 _TREE redacted_text 
3 RIN redacted_# 
3 NOTE My main database 
3 _ENV prd 
2 CORP Ancestry.com 
3 PHON 801-705-7000 
3 WWW www.ancestry.com 
3 ADDR 1300 West Traverse Parkway 
4 CONT Lehi, UT 84043 
4 CONT USA 
1 DATE 2 May 2023 
2 TIME 15:11:03 
1 GEDC 
2 VERS 5.5.1 
2 FORM LINEAGE-LINKED 
1 CHAR UTF-8
```

Anyone know where there is accurate information about Ancestry's GEDCOM7 implementation schedule? (Still listed as "tbd" for Ancestry on the FamilySearch webpage.)

Or how to make their export format dates in the expected "d MMM yyyy" format?

**Comment by clarkegj** [comment, marked as answer]
I  have spoken to our GEDCOM representative at Ancestry.  Here are my findings:
Ancestry does not standardize dates.  It accepts what ever is the entered value.  Ancestry does not yet support any version of GEDCOM 7 and hasn't set a data for implementation.  Please contact Ancestry directly about Ancestry's services.
